app/js/containers/Seller/1_SellerLocationContainer.jsx

```diff
@@ -1,16 +1,17 @@
 import React, {Component, Fragment} from 'react';
 import PropTypes from 'prop-types';
-import SellerPosition from '../../components/Seller/SellerPosition';
 import {connect} from "react-redux";
-import seller from "../../features/seller";
 
-class SellerPositionContainer extends Component {
+import SellerPosition from '../../components/Seller/SellerPosition';
+import newSeller from "../../features/newSeller";
+
+class SellerLocationContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      location: props.location
+      location: props.seller.location
     };
-    this.validate(props.location);
+    this.validate();
```

Comment:
Does this work? I know `location` is kind of global, but in the function, the arg will be undefined, no?

Reply:
This is a mistake but it seems to work...weird